graderx/graders/moss/moss.pl

```diff
@@ -164,7 +164,7 @@
 #
 # The userid is used to authenticate your queries to the server; don't change it!
 #
-$userid=631291500;
```

Comment:
I guess this shouldn't be public :sweat_smile: I'm not sure how to do this in Perl though. If you don't have time then ignore it